api/uninstall/service.go

```diff
@@ -3,24 +3,34 @@ package uninstall
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/gojekfarm/albatross/pkg/helmcli"
 	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
 
 	"helm.sh/helm/v3/pkg/release"
 )
 
+const defaultTimeout = 5 * time.Minute
+
 type Service struct {
 	cli helmcli.Client
 }
 
 // Uninstall a release according to the request provided and fails if req is incorrect.
 func (s Service) Uninstall(ctx context.Context, req Request) (Response, error) {
+	var timeout time.Duration
+	if req.Timeout == 0 {
+		timeout = defaultTimeout
+	} else {
+		timeout = time.Second * time.Duration(req.Timeout)
+	}
```

Comment:
Can we have a test where `defaulttimeout` kicks in.

Reply:
We have a test where this branch is tested [default-timeout](https://github.com/gojekfarm/albatross/pull/8/files#diff-a3759230cb828316e339f88f26f61fb768a8bf40ccd510f8656f25c2c0fc6a24R136) vs [custom-timeout](https://github.com/gojekfarm/albatross/pull/8/files#diff-a3759230cb828316e339f88f26f61fb768a8bf40ccd510f8656f25c2c0fc6a24R113)

Reply:
@punit-kulal In the case of `default-timeout` you're still passing the value externally. The value itself isn't 0. I was talking about that case.

Reply:
@buildAI  It's not being passed externally. Request is the arguement which is passed into the function.
The default timeout passed is the expected value for the mock call. 
Flow: Req (without timeout mentioned) -> expects the uninstallFlag to be initialised with defaultTimeout value -> the mock function won't be triggered if the uninstall flag struct was not initialised with defaultTimeout failing the test

Reply:
@punit-kulal Sorry misinterpreted.